go.mod

```diff
@@ -11,3 +11,5 @@ require (
 	golang.org/x/net v0.0.0-20190206173232-65e2d4e15006 // indirect
 	golang.org/x/sys v0.0.0-20190204203706-41f3e6584952 // indirect
 )
+
+go 1.13
```

Comment:
gah....my IDE keeps adding this

Reply:
I don't think is your IDE but the `go` tooling instead. I've decided to keep it there, if you're using modules you're probably on `1.13` already, if not this has no effect for you 🤷‍♂